### PR TITLE
Simple git ssh setup

### PR DESCRIPTION
**Pull request by ericzolf**
### What does this PR do?

Add new role to setup a minimal Git server
    
I used already the same approach a few times and it's optimal to quickly setup a minimal,
SSH based, Git server typically on Tower itself, for learning and demo purposes.

### How should this be tested?

The minimal example in the README is fully functional and you just need the name of a server (e.g. Tower) in your inventory, the defaults are fully functional.

### Is there a relevant Issue open for this?

n/a

### Other Relevant info, PRs, etc.

I'd be willing to put automated testing but I'd need an example/instructions to start from.

### CODE REVIEW

File: roles/git_ssh_setup/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (Tompage1994) [inline roles/git_ssh_setup/.travis.yml]: We can get rid of this file as it is not required

File: roles/git_ssh_setup/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Role Name
```
Comment (Tompage1994) [inline roles/git_ssh_setup/README.md]:
```suggestion
redhat_cop.tower_utilities.git_ssh_setup
```

File: roles/git_ssh_setup/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - hosts: tower.example.com
+      roles:
+      - git_ssh_setup
+
+We've assumed that it will be created directly on the Tower server for demonstration purposes.
+
+License
+-------
+
+BSD
```
Comment (Tompage1994) [inline roles/git_ssh_setup/README.md]:
```suggestion
MIT
```

File: roles/git_ssh_setup/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+galaxy_info:
```
Comment (Tompage1994) [inline roles/git_ssh_setup/meta/main.yml:2]: Could you change this file so it matches the format in the other roles in this repo

**Review by Tompage1994 — CHANGES_REQUESTED** [review]
I've added some comments below.

Also could you do the following:
- remove .swp file
- Check the linting issues
- Perhaps add something to the readme (or maybe even the main readme) about the relevance of having this role in this collection.

Altogether good though

File: roles/git_ssh_setup/meta/main.yml (hunk 4 above)
Comment (ericzolf) [reply roles/git_ssh_setup/meta/main.yml:2]: Done.

File: roles/git_ssh_setup/README.md (hunk 3 above)
Comment (ericzolf) [reply roles/git_ssh_setup/README.md]: Obviously, sorry.

File: roles/git_ssh_setup/README.md (hunk 2 above)
Comment (ericzolf) [reply roles/git_ssh_setup/README.md]: Done.

File: roles/git_ssh_setup/.travis.yml (hunk 1 above)
Comment (ericzolf) [reply roles/git_ssh_setup/.travis.yml]: yep, done.

**Comment by ericzolf** [comment]
>     * remove .swp file

Done and added to the .gitignore file (basically merged devel into my branch)

>     * Check the linting issues

They were gone? But perhaps I forgot to push, anyway they're gone now. The errors in the checks are not related to linting any more, they're just weird pipeline errors.
 
>     * Perhaps add something to the readme (or maybe even the main readme) about the relevance of having this role in this collection.

I described the features of the collection in the top README, I think, it's where it fits best.
 
> Altogether good though

Thanks :smile: and thanks for the constructive feedback.

**Comment by ericzolf** [comment]
I have no damn clue why the pipeline continues to fail, as I can't see any meaningful error message. On my laptop, everything looks fine:

```
$ ansible-lint -c .github/workflow-config/.ansiblelint.yml --exclude=roles/master_role_example  -v roles
$ ansible-lint -c .github/workflow-config/.ansiblelint.yml --exclude=roles/master_role_example -v roles
```
The only thing which is strange is the following but it's obviously an error of ansible-lint and the error code is still 0:

```
$ ansible-lint -c .github/workflow-config/.ansiblelint.yml --exclude=roles/master_role_example -v roles/git_ssh_setup/
Examining roles/git_ssh_setup/tasks/git_users.yml of type tasks
Examining roles/git_ssh_setup/tasks/main.yml of type tasks
Examining roles/git_ssh_setup/meta/main.yml of type meta
Examining roles/git_ssh_setup/tasks/{'file': of type tasks
WARNING: Couldn't open /home/myuser/.ansible/collections/ansible_collections/redhat_cop/tower_utilities/roles/git_ssh_setup/tasks/{'file': - No such file or directory
```

**Comment by ericzolf** [comment]
OK, found the issue thanks to @sean-m-sullivan (stupid empty line overseen).

**Review by sean-m-sullivan — APPROVED** [review]
Looks good to me, going go have Tom do a review then should be good to merge.

**Review by Tompage1994 — APPROVED** [review]
LGTM

**Pull request merged**